internal/magickeepalive/refresher.go

```
@@ -0,0 +1,83 @@
+	if next.RefreshToken == "" {
+		next.RefreshToken = current.RefreshToken
+	}
+
+	if next.TokenType == "" {
+		next.TokenType = current.TokenType
+	}
+
+	blob, err := json.Marshal(next)
+
+	if err != nil {
+		return fmt.Errorf("Failed to marshal blog: %w", err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)